### PR TITLE
Deposit in missed execution

### PR DESCRIPTION
**Pull request by kosecki123**
Deposit is transferred to a TimeNode that is executing after missed execution as well, thus added to profitability equation.

### CODE REVIEW

File: test/unit/UnitTestConfig.ts [hunk 1]
```diff
@@ -38,14 +39,14 @@ describe('Config unit tests', () => {
 
     it('Detect if config client is set', () => {
       const config = new Config({ providerUrl });
-      expect(config.clientSet()).to.be.false;
+      expect(config.clientSet()).to.be.false; // tslint:disable-line no-unused-expression
```
[the remark this answers is not in the document]
Comment (kosecki123) [reply test/unit/UnitTestConfig.ts]: we are using expect(). notation in all tests, if we are going to change to assert then we should change them all.

[the remark this answers is not in the document]
Comment (Bagaric) [reply test/unit/UnitTestConfig.ts]: We use both `expect` and `assert`.

[the remark this answers is not in the document]
Comment (Bagaric) [reply test/unit/UnitTestConfig.ts]: We even use `assert` more often.

File: index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { Config, TimeNode, StatsDB, Wallet, version } = require('./built');
+const { Config, TimeNode, StatsDB, Wallet, version } = require('./built/src');
```
Comment (Bagaric) [inline index.js]: This change creates a weird folder structure and breaks some of the functionality.

screen shot 2018-08-20 at 14 52 16

Comment (Bagaric) [reply index.js]: screen shot 2018-08-20 at 14 53 09

**Pull request merged**